src/megaclient.cpp

```diff
@@ -2088,19 +2104,21 @@ void MegaClient::exec()
 
                 // fall through
             case REQ_FAILURE:
-                if (pendingscUserAlerts->httpstatus == 200)
-                {
-                    error e = (error)atoi(pendingscUserAlerts->in.c_str());
-                    if (e == API_EAGAIN || e == API_ERATELIMIT)
-                    {
-                        btsc.backoff();
-                        pendingscUserAlerts.reset();
-                        LOG_warn << "Backing off before retrying useralerts request: " << btsc.retryin();
-                        break;
-                    }
-                    LOG_err << "Unexpected sc response: " << pendingscUserAlerts->in;
-                }
-                LOG_err << "Useralerts request failed, continuing without them";
+//TODO: uncomment this
+                //if (pendingscUserAlerts->httpstatus == 200)
+                //{
+                //    error e = (error)atoi(pendingscUserAlerts->in.c_str());
+                //    if (e == API_EAGAIN || e == API_ERATELIMIT)
+                //    {
+                //        btsc.backoff();
+                //        pendingscUserAlerts.reset();
+                //        LOG_warn << "Backing off before retrying useralerts request: " << btsc.retryin();
+                //        break;
+                //    }
+                //    LOG_err << "Unexpected sc response: " << pendingscUserAlerts->in;
+                //}
+                LOG_warn << "Useralerts request failed, continuing without them";
+
```

Comment:
REMINDER :mega: :mega: : do not resolve this discussion until the above has bee uncommented